tests/testing/unit/harness/harness.tsx

```diff
@@ -620,6 +622,42 @@ describe('harness', () => {
 			));
 		});
 
+		it('should mock icache middleware', () => {
```

Comment:
should this test be in the icache mock test suite over the harness?

Reply:
I should maybe just change the name, or use fake middleware? It's really just to test that it allows a mock at all, not to verify the icache mock functionality